### PR TITLE
feat: app containerization for local development

### PR DESCRIPTION
**Pull request by sunilthorat09**
Containerised  web app  frontend and azure functions api services. It will be used for local development with `docker-compose`.

### CODE REVIEW

**Review by nirvanatikku — APPROVED** [review]
Thank you for this enhancement!

File: README.md [hunk 1]
```diff
@@ -92,8 +94,47 @@ mkcert -install
...
+#### Configure
+
+* Generated certificate files `certificate.pfx`, `localhost+2.pem` and `localhost+2-key.pem` should be present in the project's base folder.
+* Supply the password used when generating certificates in a file `.password`.
+  
+#### Build and Run
+
+```bash
+docker compose build
+docker compose up -d
```
Comment (jbf154) [inline README.md:125]:
```suggestion
docker-compose up -d
```

File: README.md [hunk 2]
```diff
@@ -92,8 +94,47 @@ mkcert -install
...
+```
+
+Launch `https://localhost:8000`
+
+Note:
 
-### Install
+* Override environment variable values in `.env` file as needed.
+* If facing issues, try with `docker compose restart`.
```
Comment (jbf154) [inline README.md:133]:
```suggestion
* If facing issues, try with `docker-compose restart`.
```

File: README.md [hunk 3]
```diff
@@ -172,7 +221,7 @@ export base_url_logapi="https://log.api.nuance.com"
 export oauth_scope="dlg nlu tts log"
 ```
 
-Note: When deployed, these can be set per deployment environment in the Azure Portal.
+Note: When deployed, these can be set per deployment environment in the Azure Portal. Override values as needed in `.env` when using `docker compose`
```
Comment (jbf154) [inline README.md:224]:
```suggestion
Note: When deployed, these can be set per deployment environment in the Azure Portal. Override values as needed in `.env` when using `docker-compose`
```

File: README.md [hunk 4]
```diff
@@ -213,9 +262,10 @@ export sendgrid_api_key="<REPLACE_ME>"
 export sendgrid_from_email="<REPLACE_ME>"
 export sendgrid_custom_token="<REPLACE_ME>"
 ```
-
+Note: Override values as needed in `.env` when using `docker compose`
```
Comment (jbf154) [inline README.md:265]:
```suggestion
Note: Override values as needed in `.env` when using `docker-compose`
```

**Comment by nirvanatikku** [comment]
@sunilthorat09 thanks!

**Pull request merged**